test/check-machines-virtualization

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python3
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2021 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import sys
+
+# import Cockpit's machinery for test VMs and its browser test API
+TEST_DIR = os.path.dirname(__file__)
+sys.path.append(os.path.join(TEST_DIR, "common"))
+sys.path.append(os.path.join(os.path.dirname(TEST_DIR), "bots/machine"))
+
+from machineslib import VirtualMachinesCase  # noqa
+from testlib import nondestructive, test_main, wait  # noqa
+
+
+@nondestructive
+class TestMachinesVirtualization(VirtualMachinesCase):
+
+    def testVirtualizationDisabled(self):
+        b = self.browser
+        m = self.machine
+
+        m.execute("touch /tmp/cpuinfo.txt && mount --bind /tmp/cpuinfo.txt /proc/cpuinfo")
```

Comment:
Our tests do not run in a guaranteed order so this tests mount's something and the test below umounts which will only work if they are run after each other.

We should not rely on this behaviour as it's random. Our test library has a special `self.addCleanup` function which can call umount after the test is done.

So please add an `self.addCleanup(m.execute, "umount /proc/cpuinfo || true")`  the `|| true` is required in case the test fails to mount (should never happen imo)